source/runtime.hpp

```diff
@@ -348,6 +356,7 @@ namespace reshade
 		bool _show_splash = true;
 		bool _show_code_editor = false;
 		bool _show_screenshot_message = true;
+		bool _show_single_effect_splash = false;
```

Comment:
Just use the default splash via `_show_splash` I think. This overcomplicates things unnecessarily.

Reply:
I use a specific bollean var, in order to display a specific splsh message, because the standard one displays too much information when compiling just one technique. Also, it allows me to display the message for only one second instead of five (which is the case in the standard splash message)

Reply:
Maybe we can refactor it by changing the _show_splash var type and associate to it three states (OFF, DISPLAY, PREPROCESSOR_DISPLAY)